ekip/everykid/urls.py

```diff
@@ -30,36 +28,39 @@
     url(
         r'get-your-pass/educator', EducatorFormPreview(EducatorForm),
         name="educator_passes"),
-    url(r'get-your-pass/', cache_page(60*60)(TemplateView.as_view(
-        template_name="get-your-pass/index.html")), name="get_your_pass"),
+    url(r'get-your-pass/', TemplateView.as_view(
```

Comment:
Removing all this caching is making me nervous. Before this hits production, I think you need to look at how to make the CSRF stuff work with caching. I believe there's some documentation on the Django website that talks to this.